Makefile

```diff
@@ -117,9 +117,12 @@ build-cnf-tests-debug: results-html
 	PATH=${PATH}:${GOBIN} ginkgo build -gcflags "all=-N -l" -ldflags "${LINKER_TNF_RELEASE_FLAGS} -extldflags '-z relro -z now'" ./cnf-certification-test
 
 # Installs build tools and other required software.
-install-tools:
+install-tools: go.sum
 	go install "$$(awk '/ginkgo/ {printf "%s/ginkgo@%s", $$1, $$2}' go.mod)"
 
+go.sum: go.mod
+	go mod tidy
+
```

Comment:
Are these changes needed?

Reply:
yes because when the dependency is injected you need to run this command.

Reply:
The go.sum shouldn't need to be recreated at this point.  Unless I'm missing something that has to do with the new depends-on action?

Reply:
The process of injecting a change is modifying `go.mod` so `go.sum` needs to be regenerated.